examples/olaclient/README.md

```diff
@@ -0,0 +1,173 @@
+olaclient README
+
+        This is an application that reads DMX data provided by olad and
+        sets the PWM channels of a PCA9685 based on the DMX data.
+
+        The DMX data provided by olad consists of a frame of an entire
+        universe.  It is interpreted as 16x 16-bit values starting with
+        the most significant byte of LED0 at DMX channel 1 followed by
+        the least significant byte of LED0 at DMX channel 2.  The next
+        15 LED values are similarly interpreted to reside in DMX channels
+        2 - 16.
+
+        The MSBs and LSBs are combined into 16-bit values which are then
+        right-shifted by 4 bits to derive the 12-bit PWM values.
+
+        Copyright (c) 2016 - 2018 Scott Edlin
+        edlins ta yahoo tod com
+
+
+DEPENDENCIES
+
+        CMake version 3.0 or later is required to build the application.
+        A working C++ compiler is also required.  g++ 4.9.2 works well.
+
+        libPCA9685 0.6 or later is required.  olaclient is bundled in
+        versions 0.7 and later.
+
+        RASPBIAN
+
+        OLA (Open Lighting Architecture) is required.  ola 0.10.2 works
+        well.  ola 0.9.1 is in the raspbian repository:
+        http://mirrordirector.raspbian.org/raspbian/ jessie/main armhf Packages
+        however that version has not been tested.  To try it, execute:
+
+        $ sudo apt-get update
+        $ sudo apt-get install libola-dev
+
+        UBUNTU 14.04 "TRUSTY"
+
+        The ola wiki regarding Ubuntu does not cover Trusty and the Ubuntu
+        Trusty repositories do not contain ola packages.  To install ola on
+        Trusty from the ola PPA execute:
+
+        $ sudo add-apt-repository ppa:voltvisionfrenchy/ola
+        $ sudo apt-get update
+        $ sudo apt-get install libola-dev
```

Comment:
You probably want to add ola itself to the package list here, I think libola-dev pulls it in, but your code isn't very useful without it, although will compile for Travis.

Reply:
Ok, thanks.  Wasn't sure because I'm not actually deploying into a clean dev env, was just trying to fix Travis.  I also added you to the AUTHORS file for your contribution and will update the README.

Reply:
Also olaclient now has EXCLUDE_FROM_ALL so the `make` won't blow up if ola is not loaded.